src/Eleventy.js

```diff
@@ -230,12 +230,16 @@ class Eleventy {
       `Wrote ${writeCount} ${simplePlural(writeCount, "file", "files")}`
     );
 
-    let time = ((new Date() - this.start) / 1000).toFixed(2);
+    let time = ((new Date().getTime() - this.start.getTime()) / 1000).toFixed(
```

Comment:
What’s the motivation for this change? I haven’t seen any bugs from this behavior